packages/dashmate/src/listr/tasks/reindexNodeTaskFactory.js

```diff
@@ -38,125 +64,99 @@ function reindexNodeTaskFactory(
     return new Listr([
       {
         title: 'Check services are not running',
-        enabled: () => config.get('core.reindex.enable'),
-        task: async () => {
-          const isRunning = await dockerCompose.isNodeRunning(generateEnvs(configFile, config));
+        enabled: (ctx) => !ctx.isDetached,
+        task: async (ctx, task) => {
+          const isNodeRunning = await dockerCompose.isNodeRunning(generateEnvs(configFile, config));
+
+          let header;
+          if (isNodeRunning) {
+            header = 'Node found running. The node will be unavailable until reindex is complete.\n';
+          } else {
+            header = 'Node is not running. The node will be automatically started and available after reindex is complete.\n';
+          }
+
+          const agreement = await task.prompt({
+            type: 'toggle',
+            name: 'confirm',
+            header,
+            message: 'Start reindex?',
+            enabled: 'Yes',
+            disabled: 'No',
+          });
 
-          if (isRunning) {
-            throw new Error('Services is running, stop your nodes first');
+          if (!agreement) {
+            throw new Error('Operation is cancelled');
           }
         },
       },
       {
-        title: 'Set reindex mode',
-        enabled: () => config.get('core.reindex.enable'),
+        title: 'Start Core in reindex mode',
         task: async () => {
-          config.set('core.reindex.enable', 1);
+          const templatePath = 'core/dash.conf.dot';
+          const [configPath] = templatePath.split('.dot');
```

Comment:
Looks confusing. Let's define configPath first and then use it as a part of templatePath?